qemu-shutdown

```diff
@@ -1,7 +1,23 @@
 #!/bin/bash
 
+# TODO: Remove this when shutdown-deferrer is fully implemented and returns
+# proper defer status.
 sleep 60
 
+POLL_INTERVAL=5
+POLL_TIMEOUT=600
+
+# Poll shutdown-deferrer service in order to wait for proper node draining
+# before VM shutdown.
+DEFER="yes"
```

Comment:
Sorry for being late here. I think this should be set to `true` if we are checking `false` in the loop.

Also convention in bash is to name env variables with upper case, and scrip variables with lower case. That way you are sure you won't override env variable by accident.

Reply:
Regarding the value, I'm already cleaning that up here: https://github.com/giantswarm/k8s-kvm/pull/24/files - I just didn't push that PR for review yet as I haven't thoroughly tested it.

I can also change variable cases there. I have personally always used to use upper case in shell scripts, but I'm fine with lower case here as well.